corehq/warehouse/transforms/__init__.py

```diff
@@ -0,0 +1,17 @@
+from corehq.warehouse.const import (
+    APP_STATUS_FACT_SLUG,
+)
+
+
+def app_status_transform(row):
+    print row
```

Comment:
that's a good question. i had kind of wanted to keep the E, T, and L parts very distinct. one concern would be that this could prolong the strain on the main database (instead of the warehouse database) since we take longer to process each record. not sure how valid that is, and it does seem like a good place to do the python transform. i'm hesitant we'll be able to get everything we want without some python